### PR TITLE
docs: add useGenerated community data provider

### PR DESCRIPTION
**Pull request by usegen**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

_Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request._

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.

### Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### Self Check before Merge

Please check all items below before review.

-   [ ] Corresponding issues are created/updated or not needed
-   [ ] Docs are updated/provided or not needed
-   [ ] Examples are updated/provided or not needed
-   [ ] TypeScript definitions are updated/provided or not needed
-   [ ] Tests are updated/provided or not needed
-   [ ] Changesets are provided or not needed

### CODE REVIEW

**Comment by usegen** [comment]
@BatuhanW  I have opened this one with the commit convention

File: documentation/src/assets/integration-icons/usegenerated.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
...f2" d="M 227.53125 158.496094 L 239.785156 158.496094 L 239.785156 216.539062 L 227.53125 216.539062 Z M 227.53125 158.496094 " fill-opacity="1" fill-rule="nonzero" /></g></g><g clip-path="url(#ace1cfe6ba)"><g clip-path="url(#0033bebb77)"><path fill="#542cf2" d="M 191.859375 158.496094 L 202.175781 158.496094 L 202.175781 216.539062 L 191.859375 216.539062 Z M 191.859375 158.496094 " fill-opacity="1" fill-rule="nonzero" /></g></g><g clip-path="url(#ee291386c2)"><g clip-path="url(#d0e68fed1d)"><path fill="#542cf2" d="M 157.742188 158.496094 L 166.125 158.496094 L 166.125 216.539062 L 157.742188 216.539062 Z M 157.742188 158.496094 " fill-opacity="1" fill-rule="nonzero" /></g></g><g clip-path="url(#18e7b4a326)"><g clip-path="url(#a49051a6ab)"><path fill="#542cf2" d="M 125.390625 158.496094 L 132.484375 158.496094 L 132.484375 216.539062 L 125.390625 216.539062 Z M 125.390625 158.496094 " fill-opacity="1" fill-rule="nonzero" /></g></g><g clip-path="url(#38b772e930)"><g clip-path="url(#cd99c07a5a)"><path fill="#542cf2" d="M 94.664062 158.496094 L 99.824219 158.496094 L 99.824219 216.539062 L 94.664062 216.539062 Z M 94.664062 158.496094 " fill-opacity="1" fill-rule="nonzero" /></g></g>
```
Comment (BatuhanW) [inline documentation/src/assets/integration-icons/usegenerated.tsx:13]: Can we compress this one?

File: documentation/docs/getting-started/overview.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ All in all, **refine** just works _seamlessly_ with any _custom designs_ or _UI
...
+🔌 Connectors for **15+ backend services** including [REST API](https://github.com/refinedev/refine/tree/master/packages/simple-rest), [GraphQL](https://github.com/refinedev/refine/tree/master/packages/graphql), [NestJs CRUD](https://github.com/refinedev/refine/tree/master/packages/nestjsx-crud), [Airtable](https://github.com/refinedev/refine/tree/master/packages/airtable), [Strapi](https://github.com/refinedev/refine/tree/master/packages/strapi), [Strapi v4](https://github.com/refinedev/refine/tree/master/packages/strapi-v4), [Strapi GraphQL](https://github.com/refinedev/refine/tree/master/packages/strapi-graphql), [Supabase](https://github.com/refinedev/refine/tree/master/packages/supabase), [Hasura](https://github.com/refinedev/refine/tree/master/packages/hasura), [Appwrite](https://github.com/refinedev/refine/tree/master/packages/appwrite), [Firebase](https://firebase.google.com/), [Directus](https://directus.io/) and [useGenerated](https://useGenerated.com/).
```
Comment (BatuhanW) [inline documentation/docs/getting-started/overview.md:31]: Hey @usegen can we move this to `Commmunity Packages` section and remove it from here? 

https://github.com/refinedev/refine/blob/e6ca248fafc2597c6ad69e598bd21b96a1cd1a27/documentation/docs/packages/list-of-packages.md#L86

**Review by aliemir — APPROVED** [review]

**Review by BatuhanW — CHANGES_REQUESTED** [review]
Asked for minor changes, as we aren't officially supporting useGenerated, it would make more sense to include it into `Community Packages` section and remove it from overview.

Additionally, you can also create pr against awesome-refine repository and include your project there too!

https://github.com/refinedev/awesome-refine#community-packages-%EF%B8%8F

File: documentation/docs/getting-started/overview.md (hunk 2 above)
Comment (usegen) [reply documentation/docs/getting-started/overview.md:31]: @BatuhanW sure!

File: documentation/src/assets/integration-icons/usegenerated.tsx (hunk 1 above)
Comment (usegen) [reply documentation/src/assets/integration-icons/usegenerated.tsx:13]: yes

**Comment by usegen** [comment]
@BatuhanW  I have pushed the changes both for compressing the SVG9let me know if it's enough)  and moving the useGenerated from overview to `Community Packages`

File: documentation/versioned_docs/version-3.xx.xx/getting-started/overview.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Instead of being a limited set of pre-styled components, **refine** is a collect
...
+🔌 Connectors for **15+ backend services** including [REST API](https://github.com/refinedev/refine/tree/master/packages/simple-rest), [GraphQL](https://github.com/refinedev/refine/tree/master/packages/graphql), [NestJs CRUD](https://github.com/refinedev/refine/tree/master/packages/nestjsx-crud), [Airtable](https://github.com/refinedev/refine/tree/master/packages/airtable), [Strapi](https://github.com/refinedev/refine/tree/master/packages/strapi), [Strapi v4](https://github.com/refinedev/refine/tree/master/packages/strapi-v4), [Strapi GraphQL](https://github.com/refinedev/refine/tree/master/packages/strapi-graphql), [Supabase](https://github.com/refinedev/refine/tree/master/packages/supabase), [Hasura](https://github.com/refinedev/refine/tree/master/packages/hasura), [Appwrite](https://github.com/refinedev/refine/tree/master/packages/appwrite), [Firebase](https://firebase.google.com/), [Directus](https://directus.io/) and [useGenerated](https://useGenerated.com/).
```
Comment (aliemir) [inline documentation/versioned_docs/version-3.xx.xx/getting-started/overview.md:30]:
```suggestion
🔌 Connectors for **15+ backend services** including [REST API](https://github.com/refinedev/refine/tree/master/packages/simple-rest), [GraphQL](https://github.com/refinedev/refine/tree/master/packages/graphql), [NestJs CRUD](https://github.com/refinedev/refine/tree/master/packages/nestjsx-crud), [Airtable](https://github.com/refinedev/refine/tree/master/packages/airtable), [Strapi](https://github.com/refinedev/refine/tree/master/packages/strapi), [Strapi v4](https://github.com/refinedev/refine/tree/master/packages/strapi-v4), [Strapi GraphQL](https://github.com/refinedev/refine/tree/master/packages/strapi-graphql), [Supabase](https://github.com/refinedev/refine/tree/master/packages/supabase), [Hasura](https://github.com/refinedev/refine/tree/master/packages/hasura), [Appwrite](https://github.com/refinedev/refine/tree/master/packages/appwrite), [Firebase](https://firebase.google.com/) and [Directus](https://directus.io/).
```

Updated to match the current version

**Review by BatuhanW — APPROVED** [review]

**Pull request merged**